plugin/trino-tpch/src/main/java/io/trino/plugin/tpch/TpchConnectorFactory.java

```diff
@@ -82,6 +83,7 @@ public Connector create(String catalogName, Map<String, String> properties, Conn
     {
         int splitsPerNode = getSplitsPerNode(properties);
         ColumnNaming columnNaming = ColumnNaming.valueOf(properties.getOrDefault(TPCH_COLUMN_NAMING_PROPERTY, ColumnNaming.SIMPLIFIED.name()).toUpperCase(ENGLISH));
+        DoubleTypeMapping doubleTypeMapping = DoubleTypeMapping.valueOf(properties.getOrDefault(TPCH_DOUBLE_TYPE_MAPPING_PROPERTY, DoubleTypeMapping.DOUBLE.name()).toUpperCase(ENGLISH));
```

Comment:
Ugh, we should switch this connector to use Bootstrap. Is there some reason we didn't do that originally?

Reply:
Yes that seem like a good idea someone could do.